foundation/foundation-mda/src/main/resources/templates/data-delivery-data-records/spark.schema.base.java.vm

```diff
@@ -139,7 +139,8 @@ public abstract class ${record.capitalizedName}SchemaBase extends SparkSchema {
         }
 
         // Remove validation columns from valid data
-        Seq<String> columnsToDrop = JavaConversions.asScalaBuffer(validationColumns).toSeq();
+        Seq<String> columnsToDrop =
+                JavaConverters.collectionAsScalaIterableConverter(validationColumns).asScala().toSeq();
```

Comment:
I: These changes are in the generated files and it doesn't seem it has been used given the generated error in the code so add a general message in the draft node to reminder user the java17 upgrade.

Reply:
What's the error? Are we missing a pom dependency?

Reply:
No.. I think this is only to replace a deprecate class `JavaConversions.asScalaBuffer`.